libs/react-helpers/src/lib/grid/vol-cell.tsx

```diff
@@ -42,16 +42,16 @@ export const Vol = React.memo(
             backgroundColor: type === VolumeType.bid ? BID_COLOR : ASK_COLOR,
           }}
         ></div>
-        <PriceCell value={value} valueFormatted={valueFormatted} />
+        <AgPriceCell value={value} valueFormatted={valueFormatted} />
       </div>
     );
   }
 );
 
 Vol.displayName = 'Vol';
 
-export const VolCell = ({ value, valueFormatted }: IVolCellProps) => (
+export const AgVolume = ({ value, valueFormatted }: AgVolCellProps) => (
```

Comment:
chore (blocking): Doesn't appear that this is used in an ag grid only the order book